lib/api/1.1/northbound/authentication.js

```diff
@@ -0,0 +1,66 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di'),
+    express = require('express'),
+    passport = require('passport'),
+    JwtStrategy = require('passport-jwt').Strategy,
+    LocalStrategy = require('passport-local').Strategy;
+
+module.exports = authRouterFactory;
+
+di.annotate(authRouterFactory, new di.Provide('Http.Api.Auth'));
+di.annotate(authRouterFactory, new di.Inject(
+        'Auth.Services'
+        )
+);
+
+function authRouterFactory (
```

Comment:
@brianparry .This is a good one. I had updated the code to expose the it as a middleware called authMiddlewareJwt, and moved it to auth-Service.js so that it can be shared. Refer to auth-Service.js line 190.

http-service.js line 186 is an example on how to make use of the middleware. Pretty much similar to what you show on your demo.